packages/core/src/App/app.jsx

```diff
@@ -74,12 +75,14 @@ const App = ({ root_store }) => {
     }, [root_store.ui]);
 
     React.useEffect(() => {
-        const debouncedHandleResize = debounce(handleResize, 400);
-        window.addEventListener('resize', debouncedHandleResize);
+        window.addEventListener('resize', handleResize);
 
-        return () => {
-            window.removeEventListener('resize', debouncedHandleResize);
-        };
+        // const debouncedHandleResize = debounce(handleResize, 400);
```

Comment:
Why is the debouncer commented out here?
It should still be used to prevent performance issues on event listener changes for `resize`. 
Refer to this article: [https://gomakethings.com/debouncing-your-javascript-events/](https://gomakethings.com/debouncing-your-javascript-events/)

It shouldn't be related to why the iPhone X landscape doesn't work since that's more of a screen width breakpoint issue.

Reply:
oh sorry it was commented just for test to fix j6+ issue and forgot to reverted it.